SimplenoteWidgets/Widgets/NewNoteWidget.swift

```diff
@@ -0,0 +1,21 @@
+import SwiftUI
+import WidgetKit
+
+@available(iOS 14.0, *)
+struct NewNoteWidget: Widget {
+    var body: some WidgetConfiguration {
+        StaticConfiguration(kind: Constants.configurationKind, provider: NewNoteTimelineProvider()) { entry in
+            NewNoteWidgetView()
+        }
+        .configurationDisplayName(Constants.displayName)
+        .description(Constants.description)
+        .supportedFamilies([.systemSmall])
+
+    }
+}
+
+private struct Constants {
+    static let configurationKind = "NewNoteWidget"
+    static let displayName = "New Note"
+    static let description = "Create a new note instantly with one tap."
```

Comment:
1. I'm assuming we need to use NSLocalizedString? 
2. Or the above is not needed, because of SwiftUI?
2. We also need to make sure that the localization scripts pick these up

Reply:
I will need to dig into this a bit.  SwiftUI does handle the localizing a little differently, but since this isn't going into a Text view, it is a description, we may be able to do what we normally do and use NSLocalizedString.  I have done that now, but one thing to note is that our localizable files are not part of the widget extension target yet.  Would imagine that it will need to be.